lib/numberpicker.dart

```diff
@@ -30,6 +31,7 @@ class NumberPicker extends StatelessWidget {
     this.zeroPad = false,
     this.highlightSelectedValue = true,
     this.decoration,
+    this.haptics = true,
```

Comment:
In order to provide backward compability, I believe it should default to false.
```suggestion
    this.haptics = false,
```

Reply:
@MarcinusX This is very helpful .Could you please edit and merge it? Or I can create another PR to set all default values to false.

Reply:
Just disabled by default. Sorry, I forgot about this PR!